api/children/childrenRouter.js

```diff
@@ -85,22 +85,23 @@ router.delete(
   }
 );
 
-router.get('/:id/enrollments', authRequired, checkChildExist, async function (
-  req,
-  res,
-  next
-) {
-  try {
-    const { id } = req.params;
-    const enrollments = await Children.getEnrolledCourses(id);
-    res.status(200).json(enrollments);
-  } catch (error) {
-    next(error);
+router.get(
+  '/:child_id/enrollments',
+  authRequired,
+  checkChildExist,
+  async function (req, res, next) {
```

Comment:
I like the use of your checkChildExist middleware as well as the addition of authRequired 👍